thirdeye/thirdeye-frontend/app/pods/components/anomaly-stats-block/template.hbs

```diff
@@ -0,0 +1,55 @@
+{{#each anomalyStats as |card|}}
```

Comment:
Nice componentization! nit: Looks like this could be componentized further as each card essentially has the same view.